app/javascript/views/Program.tsx

```diff
@@ -35,6 +38,24 @@ const Program = () => {
       });
   }, []);
 
+  useEffect(() => {
```

Comment:
This was weirdly needed, otherwise the confetti would respawn every tick

Reply:
huh. Interesting... also... confetti???